### PR TITLE
(maint) fixes the fastcgi test for debian and older ubuntu releases

### PR DESCRIPTION
**Pull request by bmjen**
For both Debian and older versions of Ubuntu, non-default repos are required to install the libapache2-mod-fastcgi package. I think the previous code worked on Ubuntu trusty, because in newer versions of Ubuntu, multiverse is enabled by default (according to the interwebs).

### CODE REVIEW

**Comment by bmjen** [comment]
@pabelanger @nibalizer this is to replace the fastcgi PR that didn't quite work from #1302

**Comment by nibalizer** [comment]
:shipit:

**Comment by bmjen** [comment]
Tested locally on Debian 6, 7, 8 and Ubuntu 10.04, 12.04, and 14.04 all have passed.

File: spec/acceptance/vhost_spec.rb [hunk 1]
```diff
@@ -1348,7 +1379,7 @@ class { 'apache::mod::fastcgi': }
             fastcgi_dir    => '/tmp/fast',
           }
         EOS
-        apply_manifest(pp, :catch_failures => true)
+        apply_manifest(pp, :catch_failures => true, :acceptable_exit_codes => [0, 6])
```
Comment (DavidS) [inline spec/acceptance/vhost_spec.rb]: exit code 6 means changes AND failures. this doesn't sound right.

**Comment by tphoney** [comment]
:+1:

**Pull request merged**